docs/Tutorials/MathComponent/md/Tutorial.md

```diff
@@ -498,7 +497,7 @@ The implementation code does the following:
 1. Emit telemetry and events.
 
 1. Invoke the `mathOpOut` port to request that `MathReceiver`
-perform the operation.
+performs the operation.
```

Comment:
"performs" is not correct here. "perform" is correct. The subjunctive mood indicates a request. See https://www.merriam-webster.com/words-at-play/getting-in-the-subjunctive-mood. (E.g., "I requested that he go to the store" not "I requested that he went to the store")

Reply:
Thanks for sharing information with links for the accepted usage! Appreciated. I corrected them or reverted the changes to the three sections you mentioned.